lib/beaker-pe/install/pe_utils.rb

```diff
@@ -629,6 +630,13 @@ def simple_monolithic_install(master, agents, opts={})
             on(master, puppet_agent('-t'), :acceptable_exit_codes => [0,2])
           end
 
+          install_bolt_on(master)
+          on(master, puppet('module install puppetlabs-bootstrap'))
+          on master, "ssh-keygen -f ~/.ssh/id_rsa -t rsa -N ''"
+          master_pub_key = on(master, "cat ~/.ssh/id_rsa.pub").stdout
+          #create_remote_file(agents, "~/.ssh/id_rsa_master.pub", master_pub_key, {:protocol => 'rsync'})
+          create_remote_file(agents, "/root/.ssh/id_rsa.pub", master_pub_key)
+          on(agents, "cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys")
```

Comment:
I think that lines 634-639 should all be moved into beaker-task_helper as a helper method.

Reply:
Good idea. Let me try that change